.github/CODEOWNERS

```diff
@@ -1,10 +1,13 @@
+* @deepthi
 bootstrap.sh @deepthi
+/.github/ISSUE_TEMPLATE/ @frouioui
+/.github/workflows/ @frouioui
 /config/mycnf/ @deepthi @shlomi-noach @mattlord
 /docker/ @derekperkins @dkhenry @mattlord
-/examples/compose @shlomi-noach
+/examples/compose @shlomi-noach @GuptaManan100 @frouioui
 /examples/demo @sougou @mattlord
-/examples/legacy_local @deepthi
-/examples/local @rohit-nayak-ps @frouioui @mattlord
+/examples/legacy_local @deepthi @GuptaManan100 @frouioui
```

Comment:
These don't exist today.

Reply:
Now I feel stupid for editing that line instead of removing it 🤣